packages/quantic/force-app/main/default/lwc/descriptionStrengthIndicator/descriptionStrengthIndicator.html

```diff
@@ -0,0 +1,21 @@
+<template>
+    <div class="slds-grid">
+        <div class="slds-col slds-p-top_x-small">
+            <lightning-progress-ring size="large" value={progression} variant={variant}></lightning-progress-ring>
+        </div>
+        <div class="slds-col">
+            <h3 class="slds-text-heading_small slds-p-left_small h3_heading">{message}</h3>
+            <lightning-accordion allow-multiple-sections-open class="accordion">
+                <lightning-accordion-section label={helpLabel}>
+                    <div class="slds-m-left_large">
+                        <slot name="content">
+                            Since when is this problem affecting you?<br />
```

Comment:
All strings displayed, other than error strings, must be localized. You can do so by adding entries in the `CustomLabels.labels-meta.xml` file and corresponding french entries in the translations file.

Reply:
just remembered this isnt meant for this project. Sorry about the confusion.